pytorch_lightning/trainer/supporters.py

```diff
@@ -377,6 +377,11 @@ def sampler(self) -> Union[Iterable, Sequence, Mapping]:
         """Return a collections of samplers extracting from loaders."""
         return apply_to_collection(self.loaders, (DataLoader, IterableDataset), getattr, 'sampler', None)
 
+    @property
+    def batch_size(self) -> Union[Iterable, Sequence, Mapping]:
```

Comment:
what is this needed for? This should be a separate PR if we really need it (don't see a reason for that right now)

Reply:
Fine.